src/pages/workspaces/List.js

```diff
@@ -91,6 +94,18 @@ const workspaceSubmissionStatus = workspace => {
   )
 }
 
+const anyPersistentDiskRequiresMigration = (workspaces, disks) => {
+  const workspacesByNamespace = _.flow(
```

Comment:
maybe a comment here on exactly how this transforms the data? its a bit hard to follow with all the functions invoked...